packages/vanilla/src/components/global-nav/side-nav/group/module/module.scss

```diff
@@ -58,7 +58,7 @@
 }
 
 .hig__global-nav__side-nav__section__group__module__submodules {
-  padding-left: 48px;
+  padding-left: 46px;
```

Comment:
This is a bit of a Magic Number that is the sum of the `Link:before` border, icon padding and width, and title padding.